packages/rum-recorder/src/domain/segmentCollection/deflateWorker.js

```diff
@@ -14,28 +14,35 @@ function workerCodeFn() {
     const { Deflate, constants } = makePakoDeflate()
 
     let deflate = new Deflate()
+    let sizeInBytes = 0
     self.addEventListener(
       'message',
       monitor((event) => {
         const data = event.data
         switch (data.action) {
           case 'write':
-            deflate.push(data.data, constants.Z_SYNC_FLUSH)
+            const binaryData = new TextEncoder().encode(data.data)
```

Comment:
Use `TextEncoder` instead of `Blob` because better supported even though it's slower than `Blob`